### PR TITLE
small_corrections: small issues detected when compiled with ifort17.0.4

### PR DESCRIPTION
**Pull request by jvdp1**
Some small issues detected when compiled with ifort 17.0.4

### CODE REVIEW

File: src/tests/stats/test_var.f90 [hunk 1]
```diff
@@ -154,7 +154,7 @@ program test_var
     call assert( all( abs( var(d, 1, d < 11) -&
                  [20._dp / 3._dp, 20._dp / 3._dp, 0.5_dp]) < dptol))
     call assert( all( abs( var(d, 2, d < 11) -&
-                 [19.0_dp, 43._dp / 3._dp, 0.5 _dp, 0.5_dp]) < dptol))
+                 [19.0_dp, 43._dp / 3._dp, 0.5_dp, 0.5_dp]) < dptol))
```
Comment (certik) [inline src/tests/stats/test_var.f90]: How is it that this even compilers with gfortran?

Comment (jvdp1) [reply src/tests/stats/test_var.f90]: I have no idea. I tested different versions of gfortran and it compiled fine with all of them.

Comment (certik) [reply src/tests/stats/test_var.f90]: (I wonder if gfortran makes the space in `0.5 _dp` optional.)

**Comment by jvdp1** [comment]
Good. I will merge it.

**Pull request merged**